client/src/components/showMeMore/observations.jsx

```diff
@@ -0,0 +1,182 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { makeStyles, styled } from '@material-ui/core/styles';
+import ExpansionPanel from '@material-ui/core/ExpansionPanel';
+import ExpansionPanelSummary from '@material-ui/core/ExpansionPanelSummary';
+import ExpansionPanelDetails from '@material-ui/core/ExpansionPanelDetails';
+import Typography from '@material-ui/core/Typography';
+import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
+import { Grid, TablePagination } from '@material-ui/core';
+
+const useStyles = makeStyles(theme => ({
+  root: {
+    width: '100vw',
+    [theme.breakpoints.up('md')]: {
+      width: '60vw',
+    },
+  },
+
+  expansionPanel: {
+    borderBottom: '2px solid #b8b8b8',
+  },
+  summary: {
+    backgroundColor: '#bbcef9',
+  },
+  detailed: { backgroundColor: '#82a4f8' },
+}));
+const RedFont = styled(Typography)({
+  fontSize: '.9rem',
+  textAlign: 'left',
+  '& span': {
+    color: '#f64141',
+    fontWeight: 'bold',
+  },
+});
+const WhiteFont = styled(Typography)({
+  textAlign: 'left',
+  fontSize: '.9rem',
+  '& span': {
+    color: 'White',
+    fontWeight: 'bold',
+  },
+});
+
+const Observations = props => {
+  const classes = useStyles();
+
+  const { detailData, observations } = props;
+
+  const [page, setPage] = React.useState(0);
+  const [rowsPerPage, setRowsPerPage] = React.useState(4);
+
+  const handleChangePage = (event, newPage) => {
+    setPage(newPage);
+  };
+
+  const handleChangeRowsPerPage = event => {
+    setRowsPerPage(+event.target.value);
+    setPage(0);
+  };
+  return (
+    <div className={classes.root}>
+      {(detailData.length ? detailData : observations)
```

Comment:
What is this checking for? Wouldn't detailData and observations yield the same value?

Reply:
This is the case where a user swipes a `WeeklyTrackerDay` component. `detailData` will be set to that observation, so only that observation's data would be shown. I don't mind getting rid of this, but then I don't see the purpose for having the ability to swipe a `WeeklyTrackerDay` component, so I would remove that too.

Reply:
I removed `detailData` and kept `observations` @acthelemann

Reply:
I'm running this locally and swiping `WeeklyTrackerDay` does nothing.